be/src/util/uid_util.h

```diff
@@ -41,14 +42,14 @@ inline void to_hex(T val, char* buf) {
 }
 
 template <typename T>
-inline void from_hex(T* ret, const std::string& buf) {
+inline void from_hex(T* ret, std::string_view buf) {
```

Comment:
What is the size of std::string_view? why not use a const reference as input argument?

Reply:
https://quuxplusone.github.io/blog/2021/11/09/pass-string-view-by-value/